### ISSUE TITLE
Improve display of queued versus hidden dataset counts on Saved Histories view

### ISSUE THREAD

**Issue by jennaj** [galaxyproject/galaxy#5554]
Saved History "Dataset" count icons are colored very similarly (light grey) for both queued and hidden datasets. Different borders, but it is hard to detect the difference unless really zoomed in.

Full screen on a 13" laptop:

<img width="509" alt="screen shot 2018-02-16 at 12 54 39 pm" src="https://user-images.githubusercontent.com/8999735/36332883-ecb2bc1a-1328-11e8-8994-9dd1a851277d.png">

Zoom:

<img width="619" alt="screen shot 2018-02-16 at 12 55 58 pm" src="https://user-images.githubusercontent.com/8999735/36332884-ecd26524-1328-11e8-86f2-d16faa5584ea.png">

**Comment by jmchilton** [comment]
Ping @martenson since he helped with the styling on the PR that changed this #5533.

@jennaj Thanks for the quick feedback on this. Any specific advice here what the styling should look like?

Maybe the border color should be black but still dashed? Add another pixel to the depth of the border? The background should be some sort of gradient from white to gray or gray to dark gray?

**Comment by martenson** [comment]
?
![screenshot 2018-02-19 12 59 04](https://user-images.githubusercontent.com/1814954/36391483-b28997a4-1574-11e8-8d96-2517585dc32c.png)

**Comment by martenson** [comment]
with black border 
![screenshot 2018-02-19 13 00 37](https://user-images.githubusercontent.com/1814954/36391547-eaf8bfca-1574-11e8-8c68-cf1af8d33dbc.png)

(I am not a fan, I think these two states should be the least visible)

**Comment by jennaj** [comment]
@jmchilton Yes, the simplest solution would be to use a different shade of some grey. If we go with Martin's suggestion above, I would place hidden before deleted in the order.

Grey datasets have indicated queued datasets for a long time. And now the darker grey indicates deleted (deleted or purged). It would be more intuitive to have hidden datasets be some other color but this explodes the icons. Which I like but others may not. We can discuss here for later: https://github.com/galaxyproject/galaxy/issues/5562

**Comment by martenson** [comment]
@jennaj Could you please lay this out in another issue? The fixes we got into 18.01 were about speeding things up, decoupling, and fixing dataset numbers. Your insights are valuable and they deserve own issue.

**Comment by jennaj** [comment]
Ok, will do, thanks!

**Comment by jennaj** [comment]
The updated history views should cover all of this!
